base/initdefs.jl

```diff
@@ -115,14 +117,15 @@ function parse_load_path(str::String)
     return envs
 end
 
+const default_named = parse_load_path("@v#.#.#|@v#.#|@v#|@default|@!v#.#")
+
 function init_load_path(BINDIR = Sys.BINDIR)
-    if !Base.creating_sysimg
-        load_path = get(ENV, "JULIA_LOAD_PATH", "@|@v#.#.#|@v#.#|@v#|@default|@!v#.#")
-        append!(empty!(LOAD_PATH), parse_load_path(load_path))
-    end
     vers = "v$(VERSION.major).$(VERSION.minor)"
-    push!(LOAD_PATH, abspath(BINDIR, "..", "local", "share", "julia", "site", vers))
-    push!(LOAD_PATH, abspath(BINDIR, "..", "share", "julia", "site", vers))
+    stdlib = abspath(BINDIR, "..", "share", "julia", "stdlib", vers)
```

Comment:
Is it kosher to have a directory environment nested inside a depot directory? I guess nothing prevents it, but it could maybe lead to confusion.